proposals/vip_6.ts

```diff
@@ -0,0 +1,39 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_6: ProposalDescription = {
+  title: 'VIP-6: PSM Restructuring',
+  commands: [
+    {
+      target: 'feiPriceBoundPSM',
+      values: '0',
+      method: 'pauseMint()',
+      arguments: [],
+      description: 'Pause Minting on the FEI PSM'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'withdrawAllERC20ToSafeAddress(address, address)',
+      arguments: ['{feiPriceBoundPSM}', '{volt}'],
+      description: 'Remove all VOLT from FEI PSM'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'addWhitelistAddress(address)',
+      arguments: ['{daiPriceBoundPSM}'], // todo add to list of addresses
```

Comment:
what are the additional addresses that need to be added?

Reply:
This is just a reminder to populate the daiPriceBoundPSM to the mainnet addresses once deployed